DELAY_CALIBRATION: 6000
FILTER_MERGE: 1
CALIBRATION_TIME: 2
